README.md

```diff
@@ -0,0 +1,36 @@
+# Archipelago Protocol
+
+Archipelago is a protocol for creating permissionless marketplaces for buying
+and selling collections of NFTs that are priced in ETH.
+
+The key features of Archipelago markets are:
+
+- Permissionless, neutral, and decentralized at the protocol level
+- Allows exchanging NFTs for ETH or WETH
+- Off-chain bids and asks, with on-chain execution
+- Optional on-chain order approval, for smart contract participation
+- Trait bidding (e.g. a bid on all fuzzy squiggles) and
+  collection-wide floor bids.
+- Allows user-specified royalties, including both seller and buyer royalites
+
+Archipelago is organized around [Markets](./contracts/Market.sol). Each Market
+allows the trading of a particular ERC-721 NFT for ETH, and is non-upgradable.
```

Comment:
Is it sufficiently clear that by "a particular ERC-721 NFT" you mean a
token _contract_, not actually the particular NFT? Maybe add "contract"?

Reply:
Fixed.